__tests__/topic.tests.js

```diff
@@ -105,3 +105,32 @@ describe("GET /api/articles/:articles_id", () => {
       });
   });
 });
+describe("GET /api/articles", () => {
+  test("Responds with an array of all articles object", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const articles = body.articles;
+        expect(Array.isArray(articles)).toBe(true);
+      });
+  });
+  test("Responds with an array of correct data types", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        //console.log(body, "<----- GET /api response");
+        const articles = body.articles;
+        articles.forEach((article) => {
+          expect(typeof article.article_id).toBe("number");
+          expect(typeof article.article_img_url).toBe("string");
+          expect(typeof article.author).toBe("string");
+          expect(typeof article.comment_count).toBe("string");
+          expect(typeof article.created_at).toBe("string");
+          expect(typeof article.title).toBe("string");
+          expect(typeof article.topic).toBe("string");
```

Comment:
Definitely a valid way of testing properties, also take a look at toMatchObject and/or objectContaining for a way to do the same without having to repeat expect(...) over and over for each property